composer.json

```diff
@@ -16,7 +16,9 @@
         }
     ],
     "require": {
-        "scheb/two-factor-bundle": "^3.2.0",
+        "php": "^7.1.3",
+        "ext-json": "*",
+        "scheb/two-factor-bundle": "^3.2.0|^4.0.0",
```

Comment:
Is this code really still compatible with v3?

Reply:
Don't see why it isn't. I think the only relevant change is the added method to `TwoFactorProvider`, but that doesn't break BC. Or do I miss something?

Reply:
Haven't looked too deep into it. Just wanted to raise attention to it.